### PR TITLE
fix: show label for filters in filter box in explore

### PR DESCRIPTION
**Pull request by simcha90**
Bug: Doesn't show label in filters for FilterBox

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
Screen Shot 2020-07-23 at 15 56 59

After:
Screen Shot 2020-07-23 at 15 57 07

### TEST PLAN
Added test for Filter box label

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by rusackas — APPROVED** [review]
In the case of a bugfix, adding an assertion to the test would be ideal. But this looks simple enough to me to approve.

File: superset-frontend/spec/javascripts/explore/components/FilterBoxItemControl_spec.jsx [hunk 1]
```diff
@@ -26,7 +26,9 @@ import FilterBoxItemControl from 'src/explore/components/controls/FilterBoxItemC
 import FormRow from 'src/components/FormRow';
 import datasources from '../../../fixtures/mockDatasource';
 
+const TEST_LABEL = 'some label';
 const defaultProps = {
+  label: TEST_LABEL,
```
Comment (villebro) [inline superset-frontend/spec/javascripts/explore/components/FilterBoxItemControl_spec.jsx:31]: As `TEST_LABEL` isn't being used anywhere else, I don't see a reason create a `const` for it.
```suggestion
const defaultProps = {
  label:  'some label',
```

**Review by villebro — CHANGES_REQUESTED** [review]
I quickly tested this, and it appears that the change adds `Filters` as the default label for new filters. It appears to be passed down from the FilterBox control panel, where label actually refers to label of the filter section. So I don't believe this is intended behavior. I'd also propose to add an assertion to the test like @rusackas hinted at.

File: superset-frontend/spec/javascripts/explore/components/FilterBoxItemControl_spec.jsx (hunk 1 above)
Comment (simchaNielsen) [reply superset-frontend/spec/javascripts/explore/components/FilterBoxItemControl_spec.jsx]: fixed

**Comment by simchaNielsen** [comment]
@villebro Hi, fixed bug that you mentioned, @rusackas added assertion as proposed, it just was removed somehow in previous commit :)

**Review by villebro — CHANGES_REQUESTED** [review]
The `propTypes` for label needs to be added, apart from that LGTM.

**Comment by simchaNielsen** [comment]
@villebro added

**Review by villebro — APPROVED** [review]

**Pull request merged**